usr/share/rear/conf/default.conf

```
@@ -2169,6 +2169,28 @@ COPY_AS_IS_NBKDC=()
 COPY_AS_IS_EXCLUDE_NBKDC=()
 ####
 
+####
+# BACKUP=NFS4SERVER
+##
+# Note: This is for restore via NFS server, where another computer can connect to the rescue system via NFS and push
+#       the backup files onto the disks after ReaR prepared and formatted them.
+#
+#       This method has no backup functionality, only restore. And it requires interaction with an external system to push the files.
+#
+#       ReaR will wait for a signal file to appear to indicate that the file transfer is complete.
+#
+REQUIRED_PROGS_NFS4SERVER=("rpc.nfsd" "rpc.mountd" "exportfs" "ss" "nproc")
```

```suggestion
REQUIRED_PROGS_NFS4SERVER=(rpc.nfsd rpc.mountd exportfs ss nproc)
```

We only quote when we know it is needed and then the quotes serve to indicate that there might be spaces or special chars in the string.